src/main/resources/default/taglib/t/singleSelect.html.pasta

```diff
@@ -63,8 +67,16 @@
                 noMatchesText: '@i18n("template.html.selects.noMatches")',
                 noMatchesCustomEntriesDescription: '@i18n("template.html.selects.useCustomEntry")',
                 placeholderText: '@placeholder',
-                suggestionsUri: '@suggestionUri'
+                suggestionsUri: '@suggestionUri',
+                suggestionsUriBuilder: function (query) {
+                    return this.suggestionsUri + (_dependencyFieldSelect ? "/" + _dependencyFieldSelect.value : '') + '?query=' + query;
+                }
             }, <i:render name="autocompleteOptions"/>));
+        if (_dependencyField) {
+            _dependencyField.addEventListener("tokens-changed", function() {
+                autocomplete.autocomplete.loadSuggestions();
+            });
```

Comment:
**Question:** Why is this necessary? Suggestions should be loaded when the input field gains focus and it should not have focus when another input field changes (as in the user selects one of the suggestions of another field).

Reply:
It was just a thought that the dropdown opens as soon as a value is selected in the dependent field. A kind of hint for the user, but of course you're right, we can probably leave that out.